main/networking/Ansible.js

```diff
@@ -106,19 +96,10 @@ class ListenSocket {
     this.socket.bind(LISTEN_PORT, () => {
       this.logger.log(`UDP Bound to ${LISTEN_PORT}`);
     });
-    ipcMain.on('studentCodeStatus', this.studentCodeStatusListener);
-  }
-
-  studentCodeStatusListener(event, { studentCodeStatus }) {
-    if (studentCodeStatus === StudentCodeStatus.TELEOP ||
-    studentCodeStatus === StudentCodeStatus.AUTONOMOUS) {
-      this.statusUpdateTimeout = 5;
-    }
```

Comment:
As of now, is there anything in the code that updates the status? (Maybe I am not looking at the right things or it is not implemented yet?)